### ISSUE TITLE
wish: publish to debian archive

### ISSUE THREAD

**Issue by rzr** [project-chip/zap#978]
Hi, I know there is a deb package released as assets
but I'll be curious to rebuild it from source...

In longer term wouldn´t it be helpful to have zap as part of debian (and derived distros ubuntu, rapsbian etc) ?

Feel free to comment or suggest any other package/distro easly to rebuild from source.

Iĺl may share some updates

Regards

**Comment by tecimovic** [comment]
Absolutely. We just never got around to doing it. I also suspect the generated `.deb` might not be done correctly. I know the `.rpm` is not (I use Fedora, so an RPM system....), so I do wonder if the `.deb` at this point is even functional. Most Linux users at this point download the ZIP file for Linux, which is a bit sad.

If someone volunteers some time to clean up the packaging for .deb/.rpm, test it a bit, and even work towards including them in the distros, I'd be more than happy.
